config/gdi-starter-kit/containerized-computation/docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+
+services:
+  funnel:
+    image: harbor.nbis.se/test/ejprd-gdi-containerizedcomputation:1
```

Comment:
If the image is build using the Dockerfile in this folder perhaps add the build instructions here as well:
```
    build:
      context: .
      dockerfile: ./Dockerfile.dind
```

Reply:
Well, yes, the docker file is used, but also the whole funnel repo. Wouldn't those added lines mean that docker tries to build a new image and tag it with the name in `image`, instead of using the pre-built image..?
Maybe we could add the lines as comments, for documentational purposes.

Reply:
Unless we add an instruction to run `docker compose pull` that is the case indeed. Adding the lines as comments would also be ok. I think it’s good to make the link explicit.

Reply:
see https://github.com/NBISweden/ejprd/pull/24/files#diff-e5b528566e01f9c50da1f3c39e57aa9a21858bdeb8e34ec3a55790419193cd41R5-R10